packages/angular-cli/ember-cli/lib/cli/cli.js

```diff
@@ -25,6 +25,16 @@ module.exports = CLI;
 CLI.prototype.run = function(environment) {
   return Promise.hash(environment).then(function(environment) {
     var args = environment.cliArgs.slice();
+
+    if (args[0] === '--help') {
+      if (args.length === 1) {
+        args[0] = 'help';
+      } else {
+        args.shift();
+        args.push('--help');
```

Comment:
Yes, --help is an option, not a command. So remove it from the start of the array and push it to the end.